### PR TITLE
[Linting]: Lint practice stub files (7 / 8)

### PR DESCRIPTION
**Pull request by J08K**
Whooop

### CODE REVIEW

**Comment by J08K** [comment]
Yayy, PR bot is fully working.

File: exercises/practice/spiral-matrix/.meta/example.py [hunk 1]
```diff
@@ -1,11 +1,16 @@
 def spiral_matrix(size):
-    matrix = [[0]*size for row in range(size)]
-    i, j, element = 0, -1, 1
-    di, dj = [0, 1, 0, -1], [1, 0, -1, 0]
-    for x in range(2*size - 1):
-        for _ in range((2*size - x) // 2):
-            i += di[x % 4]
-            j += dj[x % 4]
-            matrix[i][j] = element
+    matrix = [[0]*size for _ in range(size)]
```
Comment (BethanyG) [inline exercises/practice/spiral-matrix/.meta/example.py:2]:
```suggestion
    matrix = [[0]*size for row in range(size)]
```

Comment (BethanyG) [reply exercises/practice/spiral-matrix/.meta/example.py:2]: I think keeping this as `row` helps clarify what is happening here.

**Review by BethanyG — CHANGES_REQUESTED** [review]
Nice work overall!  🌟   ... We just need to make sure that the stub file for `scale generator` is cleaned out.  ;-) .  The other two are suggestions for readability.  Thanks for submitting this!

File: exercises/practice/scale-generator/scale_generator.py [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+        pitches = []
+        scale = self._reorder_chromatic_scale()
+        for _, interval in enumerate(intervals):
+            pitches.append(scale[last_index])
+            last_index += self.ASCENDING_INTERVALS.index(interval) + 1
+        return pitches
+
+    def _reorder_chromatic_scale(self):
+        index = self.chromatic_scale.index(self.tonic)
+        return self.chromatic_scale[index:] + self.chromatic_scale[:index]
```
[the remark this answers is not in the document]
Comment (J08K) [reply exercises/practice/scale-generator/scale_generator.py:29]: Oh whoopsie daisy

**Review by BethanyG — APPROVED** [review]
💜  🌟

**Pull request merged**